ui/app/adapters/secret-v2.js

```diff
@@ -8,7 +8,10 @@ export default ApplicationAdapter.extend({
   _url(backend, id) {
     let url = `${this.buildURL()}/${encodePath(backend)}/metadata/`;
     if (!isEmpty(id)) {
-      url = url + encodePath(id);
+      // if path has a whitespace already encoded, return to whitespace because method encodePath will encode it.
+      // if we don't return to a whitespace it will get encoded as %20, resulting in a %2520 (% => 25).
+      let path = id.replace(/%20/g, ' ');
```

Comment:
Should we instead decode and then encode? or find where we're encoding upstream and remove it? My concern is that any other encoded values will get double-encoded as well, and then we'll end up in a completely wonky state where it's hard to get back to the original value. Also, I'd like to see some tests (and a changelog) around this before we merge

Reply:
@chelshaw let's chat about this during our sync today.  From what I can tell the URL comes into queryRecord (which then goes to the _url method) already encoded.  And ✅  on tests and changelog.